Sources/System/Internals/Constants.swift

```diff
@@ -14,7 +14,6 @@
 #if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
 import Darwin
 #elseif os(Linux) || os(FreeBSD) || os(Android)
-import CSystem
 import Glibc
 #elseif os(Windows)
 import CSystem
```

Comment:
Doesn’t this cause a warning if CSystem imported inconsistently?

Reply:
You are right, it was being imported inconsistently for Windows, for Linux it is consistent (and not needed for `Constants.swift`)

Reply:
I updated the patch so that `CSystem` is only imported as `@_implementationOnly` on Linux. I because of the differences of the APIs of Darwin, Glibc and ucrt, CSystem is only needed for internal usage on Linux, Darwin doesn't need it at all, and Windows references CSystem symbols in public APIs (or internal inlined ones).